api/graphql/makeModels.js

```diff
@@ -204,16 +204,20 @@ export default function makeModels (userId, isAdmin, apiClient) {
       fetchMany: ({
         activePostsOnly = false,
         afterTime,
+        announcementsOnly,
```

Comment:
one thing to test is because of the way we do queries in evo, do we need to add these parameters to more places. test the Recent Activity on the user profile and also the map